pylemmy/api/comment.py

```diff
@@ -41,35 +40,33 @@ class CommentView(BaseApiModel):
     creator: Person
     creator_banned_from_community: bool
     creator_blocked: bool
-    my_vote: Optional[int]
+    my_vote: Optional[int] = None
     post: Post
     saved: bool
     subscribed: SubscribedType
 
 
 class CommentResponse(BaseApiModel):
     comment_view: CommentView
-    form_id: Optional[str]
+    form_id: Optional[str] = None
     recipient_ids: List[int]
 
 
 class GetComment(BaseApiModel):
-    auth: Optional[str]
     id: int
 
 
 class GetComments(BaseApiModel):
-    auth: Optional[str]
-    community_id: Optional[int]
-    community_name: Optional[str]
-    limit: Optional[int]
-    max_depth: Optional[int]
-    page: Optional[int]
-    parent_id: Optional[int]
-    post_id: Optional[int]
-    saved_only: Optional[bool]
-    sort: Optional[CommentSortType]
-    type_: Optional[ListingType]
+    community_id: Optional[int] = None
+    community_name: Optional[str] = None
+    limit: Optional[int] = None
+    max_depth: Optional[int] = None
+    page: Optional[int] = None
+    parent_id: Optional[int] = None
+    post_id: Optional[int] = None
+    saved_only: Optional[bool] = None  # Seems not to be in API anymore
```

Comment:
```suggestion
```
can simply remove it then! I guess it was in an old version of the API, but no need to keep it

Reply:
A bit conflicted here; this is found in the official Lemmy api, but setting a default of false (hence using it in the api) causes the test to fail. This will need more study, but removing the comments for now, and adding a few more parameters from the api.